packages/env/tests/cli.test.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
```

Comment:
In other places, tests are located in the `test` folder.

Reply:
So the parent of this file should be `test` instead of `tests`?

Reply:
`packages/env/test/cli.js` - will match the file.

Reply:
Yeah, I think this pattern fits in better with this specific package structure, but if we have a convention we should enforce it. Maybe we should add a lint rule for this?

Reply:
Well, if you update the test, then Lerna will detect those changes. So we can just include another pattern to ignore when deciding whether the package was updated. In a similar way, this applied to the command which builds packages:

https://github.com/WordPress/gutenberg/blob/35e44b94ebb5215850563db573d00810124cb230/bin/packages/watch.js#L30

https://github.com/WordPress/gutenberg/blob/35e44b94ebb5215850563db573d00810124cb230/bin/packages/build.js#L128-L131

It isn't something very pressing, but it would be nice to have it all covered one way or another.

Reply:
Yeah, I realized that a few minutes ago while updating lerna ignores in the Storybook PR.

So, I changed this in one of the follow up PRs: https://github.com/WordPress/gutenberg/pull/17732/commits/e0efa055631ab66ed532c73eff1c184f92585995.